packages/store-sync/src/watchLogs.ts

```diff
@@ -109,8 +108,9 @@ export function watchLogs({ url, address, fromBlock }: WatchLogsInput): WatchLog
       debug("logs$ subscription closed, closing client");
       try {
         client?.close();
+        client?.socket?.close();
```

Comment:
i assumed the socket should automatically be closed if the client is closed, but adding this to make sure, since we're seeing many open connections in the infra metrics that abruptly drop when the indexer restarts